libsrc/creatix-emul/c39.h

```diff
@@ -0,0 +1,84 @@
+#ifndef C39_H
```

Comment:
Comment regarding the `c39...` files in general: I don't see why a target library should contain files named like the CPU it uses. Please elaborate what it means.